lms/src/lms/signature.rs

```diff
@@ -0,0 +1,318 @@
+//! Contains the [Signature] type
```

Comment:
Similar concern to [this one](https://github.com/RustCrypto/signatures/pull/801/files#r1480048307) regarding backticks (and elsewhere in this file):

```suggestion
//! Contains the [`Signature`] type
```